.github/actions/release/bump-versions/action.yml

```diff
@@ -66,4 +80,13 @@ runs:
       if: ${{inputs.release-target == 'wasm'}}
       working-directory: bindings/wasm
       run: |
-        npm version ${{ inputs.version }}
\ No newline at end of file
+        npm version ${{ inputs.version }}
+
+    - name: Bump stronghold nodejs npm package version
+      shell: bash
+      if: ${{inputs.release-target == 'wasm'}}
+      working-directory: bindings/stronghold-nodejs
+      run: |
+        npm version ${{ inputs.version }}
+        # set peer dependency version
+        echo "`jq '.peerDependencies."@iota/identity-wasm"="${{ inputs.version }}"' package.json`" > package.json
```

Comment:
Currently the peer dependency version is set with a caret (^), this sets just the version. Does that matter? I think the default is still the caret-behaviour if unspecified?

BEFORE:
```json
  "peerDependencies": {
    "@iota/identity-wasm": "^0.5.0-dev.4"
  }
```

AFTER:
```json
  "peerDependencies": {
    "@iota/identity-wasm": "0.5.0-dev.5"
  }
```

Reply:
It does matter and is very important to be without `^`, because we want to lock step versions. Without the `^` updating `@iota/identity-wasm` without updating the bindings will error, which is what we want.